keyboards/cyboard/dactyl/keymaps/singlearc_number_row/rules.mk

```diff
@@ -6,3 +6,16 @@ TAP_DANCE_ENABLE = yes
 COMBO_ENABLE = yes
 KEY_OVERRIDE_ENABLE = yes
 VIALRGB_ENABLE = yes
+
+#-----------------------------------------------------------------------------
+# sunaku
+#-----------------------------------------------------------------------------
+
+# https://docs.qmk.fm/#/feature_caps_word?id=caps-word
+CAPS_WORD_ENABLE = yes
+
+# https://docs.qmk.fm/#/custom_quantum_functions?id=deferred-execution
+DEFERRED_EXEC_ENABLE = yes
```

Comment:
This isn't needed for the `SPLIT_USB_DETECT` fix. :policeman: This just enables QMK deferred execution for [my home row mods enhancement](https://sunaku.github.io/home-row-mods.html).